benchmarks/config/base/base.json

```diff
@@ -40,7 +40,7 @@
       "type": "XSMM-DNN",
       "benchmark": "xsmm_dnn_mlp",
       "environment": {},
-      "flags": [ "100", "256", "3", "F", "64", "64", "64", "0", "1024", "1024", "1024", "1024" ],
+      "flags": [ "100", "256", "5", "F", "64", "64", "64", "0", "1024", "1024", "1024", "1024" ],
```

Comment:
Why not use 32x32 to align with the rest?

Reply:
Yeah, it would also make more sense when comparing against the "model" ones, since the compiler still doesn't have a way to propagate tile sizes from `tpp-opt` yet.